### PR TITLE
Feature/add version cmd

### PR DESCRIPTION
**Pull request by shereenElSayed**
## Description

- version command is added to hop.
- test point is added, it does not check on the printed version since it can be updated anytime for any of the dependencies and we don't want to keep updating them.
- documentation is updated

## Checklist

* [x] All new functions and classes are documented and adhere to Google doc style (3.8.3-3.8.6 of [this](http://google.github.io/styleguide/pyguide.html#383-functions-and-methods) document)
* [x] Add/update sphinx documentation with any relevant changes.
* [x] Add/update pytest-style tests in `/tests`, ensuring sufficient code coverage.
* [x] `make test` runs without errors.
* [x] `make lint` doesn't give any warnings.
* [x] `make format` doesn't give any code formatting suggestions.
* [x] `make doc` runs without errors and generated docs render correctly.
* [x] Check that CI pipeline run on this PR passes all stages.
* [ ] Review signoff by at least one developer.

NOTE: If this PR relates to a release, open and reference an issue with the Release checklist template.

### CODE REVIEW

**Comment by shereenElSayed** [comment]
Thanks for the review, I implemented the changes for `make lint` but I did not find any changes listed in the format.

**Comment by myNameIsPatrick** [comment]
@shereenElSayed, that's fine, I should update the PR template now that `make format` has been changed. So now it'll fix everything that's causing issues in `make lint` rather than give additional suggestions, so you don't need to do it manually. It all looks good now, I'll approve the PR.

File: tests/test_version.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from hop import version
+
+
+def test_version():
+    process = subprocess.run(
+        ['hop', 'version'], stdout=subprocess.PIPE, universal_newlines=True)
+
+    versions = process.stdout
+    packages = version.get_packages()
+    assert any(package in versions for package in packages)
```
Comment (spenczar) [inline tests/test_version.py:16]: Shouldn't this be `all` instead of `any`?

Comment (shereenElSayed) [reply tests/test_version.py:16]: I didn't see your comment before I merge.
Well, yeah it seems that all is better. I will fix it and remerge.
Nice catch! Thanks

**Pull request merged**